### PR TITLE
Issue 6

### PR DESCRIPTION
**Pull request by hoanguyen071710**
Modify updater fix conn bug

### CODE REVIEW

File: faker/transaction_updater.py [hunk 1]
```diff
@@ -3,24 +3,23 @@
 import datetime
 import os
 
-
```
Comment (boushphong) [inline faker/transaction_updater.py:6]: New line here

File: faker/transaction_updater.py [hunk 2]
```diff
@@ -33,11 +32,12 @@ def user_stock_matching(transaction_count):
...
+def insert_into_db(conn, transaction_count):
+    result_user, stock_info, transaction_count = get_info(conn, transaction_count)
+    transaction_query = ','.join(user_stock_matching(result_user, stock_info, transaction_count))
     insert_query = f'''
     INSERT INTO stock_db.Transaction (user_id, stock_id, quantity, transaction_date)
     VALUES {transaction_query}
     '''
     conn.execute(text(insert_query))
-    conn.commit()
+    conn.commit()
```
Comment (boushphong) [inline faker/transaction_updater.py:43]: new line at the EOF.

File: faker/user_updater.py [hunk 3]
```diff
@@ -19,18 +19,17 @@ def update_rate():
 
 
 # Update these users info
```
Comment (boushphong) [inline faker/user_updater.py:21]: No need to comment. function name is specific enough

File: faker/user_updater.py [hunk 4]
```diff
@@ -19,18 +19,17 @@ def update_rate():
...
-    update_user = update_rate()
+    update_user = update_rate(conn)
     for usr in update_user:
         user_id = usr[0]
         new_email = fake.email()
         update_query = f"UPDATE stock_db.User SET email = '{new_email}' WHERE user_id = {user_id}"
         conn.execute(text(update_query))
-        conn.commit()
 
 # Create new users
```
Comment (boushphong) [inline faker/user_updater.py:31]: No need to comment. function name is specific enough

**Comment by boushphong** [comment]
Can you test these functions on lambda?

**Review by boushphong — APPROVED** [review]

**Review by DangMinhHao — APPROVED** [review]

**Pull request merged**